### ISSUE TITLE
Support for video recording

### ISSUE THREAD

**Issue by aseef17** [ricnaaru/adv_camera#5]
Will support be added for video recording?

**Comment by ricnaaru** [comment]
yeah, it's possible, but i think not in the near future, because i have so much on my plate currently :)

**Comment by jvinicius66** [comment]
Has a prevision to support a record video?
Thanks.
